simuleval/data/dataloader/s2t_dataloader.py

```diff
@@ -79,40 +87,52 @@ def get_source_audio_path(self, index: int):
 
     @classmethod
     def from_files(
-        cls, source: Union[Path, str], target: Union[Path, str]
+        cls,
+        source: Union[Path, str],
+        target: Union[Path, str],
+        tgt_lang: Union[Path, str],
     ) -> SpeechToTextDataloader:
         with open(source) as f:
             source_list = [line.strip() for line in f]
         with open(target) as f:
             target_list = [line.strip() for line in f]
-        dataloader = cls(source_list, target_list)
+        with open(tgt_lang) as f:
+            tgt_lang_list = [line.strip() for line in f]
+        dataloader = cls(source_list, target_list, tgt_lang_list)
         return dataloader
 
     @classmethod
     def from_args(cls, args: Namespace):
         args.source_type = "speech"
         args.target_type = "text"
-        return cls.from_files(args.source, args.target)
+        return cls.from_files(args.source, args.target, args.tgt_lang)
 
 
 @register_dataloader("speech-to-speech")
 class SpeechToSpeechDataloader(SpeechToTextDataloader):
     @classmethod
     def from_files(
-        cls, source: Union[Path, str], target: Union[Path, str]
+        cls,
+        source: Union[Path, str],
+        target: Union[Path, str],
+        tgt_lang: Union[Path, str, None] = None,
     ) -> SpeechToSpeechDataloader:
         with open(source) as f:
             source_list = [line.strip() for line in f]
         with open(target) as f:
             target_list = [line.strip() for line in f]
-        dataloader = cls(source_list, target_list)
+        tgt_lang_list = []
+        if tgt_lang is not None:
+            with open(tgt_lang) as f:
+                tgt_lang_list = [line.strip() for line in f]
+        dataloader = cls(source_list, target_list, tgt_lang_list)
```

Comment:
Great work on catching that S2S dataloader is missing this part and adding it!

Reply:
Thanks